pkg/operator/starter.go

```diff
@@ -41,6 +42,18 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+type encryptionProvider []schema.GroupResource
```

Comment:
@p0lyn0mial I suppose this could have been in library-go next to the interface as `SimpleProvider`

Reply:
yes, thx :)